features/providers/passported_journey.feature

```diff
@@ -0,0 +1,223 @@
+    When I choose "Yes"
+    And I click 'Save and continue'
+    Then I should be on a page showing "Domestic abuse summary"
+    And I choose option "Application merits task opponent warning letter sent True field"
+    Then I choose option "Application merits task opponent police notified True field"
+    Then I choose option "Application merits task opponent bail conditions set True field"
+    Then I fill "Bail conditions set details" with "Foo bar"
+    Then I fill "Police notified details" with "Foo bar"
+    Then I click 'Save and continue'
+    And I should not see "Client received legal help"
+    Then I should be on a page showing "Provide a statement of case"
+    Then I fill "Application merits task statement of case statement field" with "Statement of case"
+    When I upload an evidence file named 'hello_world.pdf'
+    Then I should not see "There was a problem uploading your file"
+    Then I should be on a page showing "hello_world.pdf"
+    Then I should be on a page showing "UPLOADED"
+    Then I click 'Save and continue'
+    Then I should be on the 'merits_task_list' page showing 'Chances of success\nNOT STARTED'
+    When I click link 'Chances of success'
+    Then I should be on a page showing "Is the chance of a successful outcome 50% or better?"
+    Then I choose "No"
+    Then I click 'Save and continue'
+    Then I should be on a page showing "What is the chance of a successful outcome?"
+    Then I choose "Borderline"
+    Then I fill "Success prospect details" with "Prospects of success"
+    Then I click 'Save and continue'
+    Then I should be on the 'merits_task_list' page showing 'Chances of success\nCOMPLETED'
+    Then I click 'Save and continue'
+    Then I should be on a page showing "Check your answers"
+    And I click Check Your Answers Change link for 'Statement of Case'
+    Then I enter the application merits task statement of case statement field 'This is some test data for the statement of case'
+    Then I click 'Save and continue'
+    Then I should be on a page showing "Check your answers"
+    Then I should be on a page showing "hello_world.pdf (15.7 KB)"
+    And the answer for 'Statement of case' should be 'This is some test data for the statement of case'
+    Then I click 'Save and continue'
+    And I should be on a page showing "Confirm the following"
+    Then I select the first checkbox
+    Then I click 'Save and continue'
+    Then I should be on a page showing "Review and print your application"
+    Then I click 'Submit and continue'
+    Then I should be on a page showing "Application complete"
+    Then I click 'View completed application'
+    Then I should be on a page showing "Application for civil legal aid certificate"
+    Then I should be on a page showing "PASSPORTED"
+
+   @javascript @vcr
+  Scenario: I am instructed to use CCMS on the passported journey with an applicant does not receive benefits
```

Comment:
this scenario appears to not match what its doing. it appears to be a non-passported journey testing a citizen email address change?!